components/org.wso2.carbon.stream.processor.statistics/src/main/java/org/wso2/carbon/stream/processor/statistics/internal/OperatingSystemMetricSet.java

```diff
@@ -74,12 +74,17 @@ protected void start(BundleContext bundleContext) {
     /**
      * Get the MBean name from the deployment yaml and get access to the MBean.
      */
-    public OperatingSystemMetricSet() {
+    public OperatingSystemMetricSet() {}
```

Comment:
formatting issues

Reply:
Fixed in 06dcfec1cfddb682d30ed864a9f2c63ca84a85b7